### DISCUSSION TITLE
Is Concourse project still... alive?

### DISCUSSION THREAD

**Discussion by robinhuiser** [concourse/concourse#8782] [General]
Hi all,

I have not seen any recent activity on the Concourse source - should we get worried?
Is this project moving to the attick?

Please let me know,
Robin

**Comment by jhohiii** [comment]
I also want to know about this.  We are waiting on the CF authentication fix for the problem that was introduced in 7.9.1 - about 4 months ago.

**Comment by gsaslis** [comment]
@robinhuiser from a quick look, https://github.com/concourse/concourse/commits/master shows plenty of activity, no?

what am I missing?

**Reply by jhohiii** [reply]
Just a release for 7.9.2 as far as I'm concerned.

Typically, bug-fix patch releases come out much sooner than 3-4 months.

**Reply by robinhuiser** [reply]
Hi @gsaslis - there is indeed commit activity on master - I am just badly used every 2-3 months there are new releases (as I am maintaining the ARM64 binaries https://github.com/robinhuiser/concourse-arm64 and https://hub.docker.com/repository/docker/rdclda/concourse).

Is there a release plan or schedule available?

**Reply by gsaslis** [reply]
well, I am only new to this forum, so I didn't have this context (which is why I was also asking what I missed - thanks for sharing!) 

I was just replying to a post that wonders if a project with - what I'd call pretty healthy commit activity on its main branch is "alive". That's all. ;)
